ow_plexil/src/plans/InitializeLander.plp

```diff
@@ -0,0 +1,25 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Reset pan and tilt angles to 0 and stow the arm (if not already there)
+
+#include "plan-interface.h"
+
+InitializeLander:
+{
+  InOut Boolean initialize;
+
+  log_info ("Initializing lander...");
+
+  log_info ("Resetting pan angle to 0...");
+  SynchronousCommand pan_antenna (0);
+  log_info ("Resetting tilt angle to 0...");
+  SynchronousCommand tilt_antenna (0);
```

Comment:
Minor point, and not intuitive, but it's better to do the tilt before the pan.  This is because pan can be potentially obstructed by the lander body, but tilt to 0 is always unrestricted.